AppGenesis/View/ContentView.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  ContentView.swift
+//  AppGenesis
+//
+//  Created by LT on 1/28/24.
+//
+
+import SwiftUI
+
+struct ContentView: View {
+  @State private var redSliderValue   = 99.0
+  @State private var greenSliderValue = 40.0
+  @State private var blueSliderValue  = 75.0
+  @State private var defaultColor = Color(red: (99.0/255), green: (40.0/255), blue: (75.0/255))
+  var body: some View {
+    
+    
+    VStack {
+      
+      Text("Color Picker")
+        .font(.title)
+      
+      RoundedRectangle(cornerRadius: 5)
+        .fill(defaultColor)
+        .frame(width: 350, height: 300)
+      
+      Text("Red")
+      HStack {
+        Slider(value: $redSliderValue, in: 1.0...100.0)
+        Text(String(Int(redSliderValue)))
+      }
+      Text("Green")
+      HStack {
+        Slider(value: $greenSliderValue, in: 1.0...100.0)
+        Text(String(Int(greenSliderValue)))
+      }
+      Text("Blue")
+      HStack {
+        Slider(value: $blueSliderValue, in: 1.0...100.0)
+        Text(String(Int(blueSliderValue)))
+      }
+      
```

Comment:
Sliders and slider values are touching the screen, can you add some padding so that they can appear with some gap with screen.